docs/HowToBuildFromSource.md

```diff
@@ -2,21 +2,17 @@
 
 If you want to use the latest version of ONE-vscode, let's do the following:
 
-## STEP1. Install vsce
+## STEP1. Install prerequisites
 
-[vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) is `Visual Studio Code Extension`.
-- You should install [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) before installing vsce
-
-```console
-$ npm install -g vsce
-```
+You should install [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). 
 
 ## STEP2. Clone repository
 
 git clone `ONE-vscode`
 
 ```console
 $ git clone https://github.com/Samsung/ONE-vscode.git
+$ cd ONE-vscode
 ```
 
 ## STEP3. Create vsix package
```

Comment:
Could you add explicit commands here? 
Let's prevent others from being confused by this STEP by adding more explanation. 

---




To install required node package dependencies, simply run `npm install`.

```
$ npm install
```

To create vsix package, run `vsce package`
```
$ vsce package
```